build.sh

```diff
@@ -19,6 +19,10 @@ set -o nounset
 set -o pipefail
 
 readonly IMAGE_TAG=v1.0.5
+
+# The image goes into two repoistories. quay.io/helmpack/chart-testing is used
```

Comment:
repoistories -> repositories

Reply:
I just learned I didn't have spell check turned on for shell files. Thanks.